src/Device/RecordedFlight.hpp

```diff
@@ -51,5 +51,5 @@ struct RecordedFlightInfo : FlightInfo {
   } internal;
 };
 
-class RecordedFlightList : public StaticArray<RecordedFlightInfo, 512u> {
+class RecordedFlightList : public StaticArray<RecordedFlightInfo, 128u> {
```

Comment:
This is an unrelated change which I don't understand, and your commit message doesn't mention it.

Reply:
This is in response to your concern that downloading a list of 512 flights might take very long, in the discussion on the issue #1237.
It is mentioned in the commit message that the max list length is reduced from 512 to 128.

Reply:
True, you can read the commit message that way (I didn't), but what it does is reduce the number of flights for ALL drivers, which is out of this commit's scope. I think that reducing the number of flights again is a good idea now that you've found a new approach, but this global change deserves a separate commit, instead of being mixed in this one.